index.html

```diff
@@ -1,63 +1,147 @@
+      <hr>
+      <ul class="lang-sec">
+
+      <li>
+        <div>
+
+          <img class="image1" src="./placeholder.png" alt="image placeholder">
+        </div>
+          <h2 class="post">Multi-Post Stories</h2>
+              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups
+                required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <ul class="lang">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
+            </li>
+      <li>
+        <div>
+
+          <img class="image1" src="./placeholder.png" alt="image placeholder">
+        </div>
+          <h2 class="post">Multi-Post Stories</h2>
+              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups
+                required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <ul class="lang">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
+            </li>
+      <li>
+        <div>
+
+          <img class="image1" src="./placeholder.png" alt="image placeholder">
+        </div>
+          <h2 class="post">Multi-Post Stories</h2>
+              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups
+                required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <ul class="lang">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
+            </li>
+      <li>
+        <div>
+
+          <img class="image1" src="./placeholder.png" alt="image placeholder">
+        </div>
+          <h2 class="post">Multi-Post Stories</h2>
+              <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups
+                required. has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+              <ul class="lang">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+              </ul>
+              <button class="button" type="button">See Project</button>
```

Comment:
- [ ] The button should have the button styling: Active, Hover, and Pressed - Please add it all buttons. [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#:~:text=Implement%20the%20button%20interactions%20(enable%2C%20hover%2C%20etc.).)
![image](https://user-images.githubusercontent.com/38635158/148649374-a4ec3c30-726b-4a3c-bcfd-3ae407a05f9e.png)

Reply:
Addressed 🔥